internal/v4/auth_test.go

```diff
@@ -123,7 +123,7 @@ func (s *commonSuite) testMacaroonAuth(c *gc.C, p httptesting.JSONCallParams) {
 		"is-authenticated-user ",
 	})
 	checkedCaveats = nil
-	c.Assert(cookieJar.cookieURLs, gc.DeepEquals, []string{"http://somehost/"})
+	c.Assert(cookieJar.cookieURLs, gc.DeepEquals, []string{"http://somehost/v4/log"})
```

Comment:
This is kind of bad though right, because the cookie is per URL path. So we might set the cookie for api.jujucharms.com/charmstore/v4/~user/trusty/charm/meta/any  but it won't be set for api.jujucharms.com/charmstore/v4/~user/trusty/charm/icon.svg